examples/escrow/Forc.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["Braqzen"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "escrow"
+
+[dependencies]
```

Comment:
For your Forc manifest files, add a local `std` dependency. That will ensure that it's actually using the local branch and therefore actually tests the project against the changeset.

You might need to regenerate the lockfiles.